gnomad/resources/resource_utils.py

```diff
@@ -159,6 +159,38 @@ def import_resource(self, overwrite: bool = True, **kwargs) -> None:
         )
 
 
+class VariantDatasetResource(BaseResource):
+    """
+    A Hail VariantDataset resource.
+    :param path: The VariantDataset path (typically ending in .vds)
+    :param import_args: Any sources that are required for the import and need to be kept track of and/or passed to the import_func (e.g. .vcf path for an imported VCF)
+    :param import_func: A function used to import the VariantDataset. `import_func` will be passed the `import_args` dictionary as kwargs.
+    """
+
+    expected_file_extensions: List[str] = [".vds"]
+
+    def vds(self, force_import: bool = False) -> hl.vds.VariantDataset:
+        """
+        Read and return the Hail VariantDataset resource.
+        :return: Hail VariantDataset resource
+        """
+        if self.path is None or force_import:
+            return self.import_func(**self.import_args)
+        else:
+            return hl.read_vds(self.path)
```

Comment:
I think this is actually:
```suggestion
            return hl.vds.read_vds(self.path)
```